core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -1134,11 +1133,11 @@ private[kafka] class Processor(val id: Int,
         // Try unmuting the channel. If there was no quota violation and the channel has not been throttled,
         // it will be unmuted immediately. If the channel has been throttled, it will unmuted only if the throttling
         // delay has already passed by now.
-        handleChannelMuteEvent(send.destination, ChannelMuteEvent.RESPONSE_SENT)
-        tryUnmuteChannel(send.destination)
+        handleChannelMuteEvent(send.destinationId(), ChannelMuteEvent.RESPONSE_SENT)
+        tryUnmuteChannel(send.destinationId())
```

Comment:
How about keeping "parameterless"? fewer changes and more clear.

Reply:
Looks like the IDE added this when I renamed it or something. Fixed.